modules/gatk_rnaseq/1.0/envs/bcftools-1.10.2.yaml

```diff
@@ -0,0 +1,36 @@
+name: test-bcftools
```

Comment:
This (and another environment file) should be rather in lcr-modules/envs/ folder, and symlinked in the module. This will make it easier to reuse same environments across modules, therefore reducing the need to build multiple environments. There is already an environment for bcftools with the same version, so you can just as well symlink `lcr-modules/envs/bcftools/bcftools-1.10.2.yaml` to this module

Reply:
Done